runtime/oti/j9.h

```diff
@@ -323,7 +323,11 @@ static const struct { \
 /* Macros for ValueTypes */
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
 #define J9_IS_J9CLASS_VALUETYPE(clazz) J9_ARE_ALL_BITS_SET(clazz->classFlags, J9ClassIsValueType)
+#define J9_IS_OBJECT_OR_VALUETYPE(char) (('L' == char) || ('Q' == char))
```

Comment:
please use a different name for the parameter, eg. `sigChar` of `character`. Some editors may get confused by the usage of `char` here